### PR TITLE
 some time need deep copy

### PR DESCRIPTION
**Pull request by loonghao**
some time need deep copy

### CODE REVIEW

**Comment by loonghao** [comment]
Hi @mewwts , Can you help me, how can fix coverage check problem? Thanks

**Comment by Ohjeah** [comment]
You need to add a test for your newly added deepcopy method.

**Comment by loonghao** [comment]
@Ohjeah Thanks!

**Comment by mewwts** [comment]
Hi @loonghao thanks for submitting this PR!
Is there any reason you prefer `my_addict_instance.deepcopy()` over `copy.deepcopy(my_addict)`?

**Comment by loonghao** [comment]
Hi @mewwts `Dict()` more object-oriented? Or have some potential risk?

File: addict/addict.py [hunk 1]
```diff
@@ -101,8 +104,8 @@ def update(self, *args, **kwargs):
         other.update(kwargs)
         for k, v in other.items():
             if ((k not in self) or
-                (not isinstance(self[k], dict)) or
-                (not isinstance(v, dict))):
```
Comment (mewwts) [inline addict/addict.py]: Could you please revert the two changes here?

File: test_addict.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
-import json
 import copy
-import unittest
+import json
 import pickle
+import unittest
+
```
Comment (mewwts) [inline test_addict.py]: Perhaps leave imports as they were?

File: test_addict.py [hunk 3]
```diff
@@ -379,7 +406,7 @@ def test_parent_key_item(self):
             a[1].x = 3
         except Exception as e:
             self.fail(e)
-        self.assertEquals(a, {'keys': {'x': 1}, 1: {'x': 3}})
+        self.assertEqual(a, {'keys': {'x': 1}, 1: {'x': 3}})
```
Comment (mewwts) [inline test_addict.py]: You can revert this change too.

**Comment by loonghao** [comment]
@mewwts Hi, Thank you feedback. I have already fixed it.

**Comment by mewwts** [comment]
Thanks @loonghao
